include/deal.II/fe/fe_q_base.h

```diff
@@ -157,6 +157,12 @@ class FE_Q_Base : public FE_Poly<dim, spacedim>
     const RefinementCase<dim> &refinement_case =
       RefinementCase<dim>::isotropic_refinement) const override;
 
+  /**
+   * These elements support mass lumping, so this function always returns true.
```

Comment:
You might want to prefix this with a @copydoc so that a reader gets to see what this function is about, before telling them what this concrete function returns. Same for the other implementations.